### ISSUE TITLE
Linphone Is Phoning Home On A Clean Installation

### ISSUE THREAD

**Issue by thegranddesign** [BelledonneCommunications/linphone-android#1656]
**Describe the bug**
On a fresh installation of Linphone, it is phoning home to the linphone servers.  I haven't dug into exactly what it's sending yet but I can only assume it's partially via the "contacts sync" described here: https://github.com/BelledonneCommunications/linphone-android/issues/1354

There is absolutely no excuse for the default being sending _any_ data to Linphone servers for any reason.  This should be explicitly _opt-in_.  Additionally if the user wishes to disable this ping, the toggle is not tappable until AFTER you give the app access to your contacts.

The following URLs are being pinged despite my app having NO Linphone servers configured:

```
lime.linphone.org
sip1.linphone.org
stun.linphone.org
_stun._udp.stun.linphone.org
```

**Expected behavior**
Only the servers I have explicitly configured or opted in to should be sent any data from the app.

**Please complete the following information**
 - Version of the App 4.6.5
 - Version of the SDK ?

**Comment by Viish** [comment]
Hi,

You are right, this was simply a mistake in how default values were handled, thanks for reporting it.
I fixed it in a464e31f and e8ad1e03 for both release/4.6 and master branches.
It will however not affect configuration for already installed apps.

Also, just so you know, we don't do anything with phone numbers sent to our server, it's merely a tool for long term presence feature.
You can check our privacy policy here: https://linphone.org/privacy-policy.

Cheers,

**Comment by thegranddesign** [comment]
@Viish so the only way to fix it is to uninstall the app and reinstall?

**Comment by Viish** [comment]
You could do it by modifying the config file directly on your phone if it's rooted but otherwise yes.

**Comment by thegranddesign** [comment]
@Viish I just downloaded the most recent release. I deleted my SIP configuration and re-added it.  While it is no longer phoning home to `sip.linphone.org`, it's still phoning home to `lime.linphone.org`.  Again, there should be NO connections to any server I don't explicitly configure.

**Comment by Viish** [comment]
Can you add logs please?

**Comment by thegranddesign** [comment]
I can't see a way to view the logs without uploading them to you all directly.  I want to view the logs before I send them.  I also didn't see them in the Linphone application directory.

If it helps, the connection attempt to `lime.linphone.org` happens any time I tap the "connected" icon in the header and it reattempts the connection to my SIP server.

If you can tell me how to access the logs on my device so I can inspect them before I send, I'll shoot em over to you.

**Comment by thegranddesign** [comment]
Also, can you reopen this issue until it's fully resolved please?

**Comment by Viish** [comment]
You can install a file transfer server script on a computer and change the upload logs URL to it: https://gitlab.linphone.org/BC/public/flexisip-http-file-transfer-server
Otherwise on your device the logs are named `linphone1.log` and `linphone2.log`, you can find them in the app directory: `adb shell run-as org.linphone.debug ls -lh files/` (remove `.debug` if you installed it from the Store).

In the meantime, can you send me the content of your config file as well please?
`adb shell run-as org.linphone.debug cat files/.linphonerc`. You can send it by mail to me directly: `sylvain.berfini@belledonne-communications.com`.

**Comment by Viish** [comment]
> I deleted my SIP configuration

When you say that, do you mean you removed the account in the app or that you uninstalled / reinstalled the app?

I just tried again and I don't reproduce your issue on a fresh install.

**Comment by thegranddesign** [comment]
I removed the account not the app.

**Comment by thegranddesign** [comment]
It seems like completely deleting the app and reinstalling it fixed the problem.  It seems kind of crazy that such extreme lengths needed to have gone through in order to fix that issue.

**Comment by Viish** [comment]
It's not that crazy. Making the change for existing users may result in broken configuration, I don't want to risk that.

**Comment by Viish** [comment]
Just so you know the requests to the LIME server for non sip.linphone.org users have been fixed in master branch (4.7).
